document/CoreFormat.mdk

```diff
@@ -994,9 +1071,27 @@ This allows tools to include information about the result that is not explicitly
 
 ### General { #location-General }
 
-A `location` object specifies the location where a static analysis tool produced a result.
+A `location` object specifies the location where an analysis tool produced a result.
 A `location` object should specify the physical location (&sect;[#physicalLocation]) of the result,
-and it may also specify the logical location (&sect;[#location-logicalLocation]) of the result.
+and it may also specify the logical location (&sect;[#location-fullyQualifiedLogicalName]) of the result.
+
+A logical location specifies a programmatic construct, for example, a class name or a function name,
+without specifying the programming artifact within which that construct occurs.
+
+~ IsoNote
+There are two reasons to include logical locations in the SARIF format in addition to physical locations:
+
+1. Logical location information is an important contributor to fingerprinting scenarios,
+    because it is more resilient to changes in source code than are line locations.
```

Comment:
i'd say 'tends to be more resilient' or 'it is typically more resilient'. if someone is on a namespace renaming scrub, line locations will remain constant but all your logical names may change...